semantic_conventions/trace/messaging.yaml

```
@@ -48,26 +77,31 @@ groups:
         type: string
         brief: 'Connection string.'
         examples: ['tibjmsnaming://localhost:7222', 'https://queue.amazonaws.com/80398EXAMPLE/MyQueue']
-      - id: message_id
-        type: string
-        brief: 'A value used by the messaging system as an identifier for the message, represented as a string.'
-        examples: '452a7c7c7c7048c2f887f61572b18fc2'
-      - id: conversation_id
-        type: string
-        brief: >
-          The [conversation ID](#conversations) identifying the conversation to which the message belongs,
-          represented as a string. Sometimes called "Correlation ID".
-        examples: 'MyConversationId'
-      - id: message_payload_size_bytes
-        type: int
-        brief: >
-          The (uncompressed) size of the message payload in bytes.
-          Also use this attribute if it is unknown whether the compressed or uncompressed payload size is reported.
-        examples: 2738
-      - id: message_payload_compressed_size_bytes
+      - id: batch.size
         type: int
-        brief: 'The compressed size of the message payload in bytes.'
-        examples: 2048
+        brief: The number of messages sent, received, or processed in the scope of the batching operation.
+        requirement_level:
+          conditionally_required: >
+            If available within the messaging system, and only if the span describes operations that operate with message batches.
```

```suggestion
            If available within the messaging system, and only if the span describes operations that involve a batch of messages.
```
Maybe? I mostly wanted to avoid:  "operations that operate"